lib/rmt/cli/export.rb

```diff
@@ -1,17 +1,17 @@
 class RMT::CLI::Export < RMT::CLI::Base
 
-  desc 'settings PATH', 'Store repository settings at given path'
-  def settings(path)
+  desc 'data PATH', 'Store SCC data in files at given path'
```

Comment:
I suppose there a specific reason to swap the order?

Reply:
![commit_comment](https://user-images.githubusercontent.com/5372947/34211020-20e40674-e598-11e7-9df7-e9576bc95e46.png)

Reply:
Ah yes, my reading skills are still exceptional.